packages/appstore/src/components/modals/real-wallets-upgrade/components/content-with-link/content-with-link.scss

```diff
@@ -0,0 +1,94 @@
+.content-with-link {
+    height: auto;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+
+    @include mobile {
+        flex-direction: column-reverse;
+    }
+
+    &--has-fork {
+        height: 100%;
+    }
+
+    &--right-forked {
+        flex-direction: row;
+
+        @include mobile {
+            flex-direction: column;
+        }
+    }
+
+    &__content {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        gap: 1.6rem;
+        margin: 0rem;
+        margin-left: 1.6rem;
```

Comment:
```suggestion
        margin: 0rem 0rem 0rem 1.6rem;
```

Reply:
Hi @aizad-deriv 😄, I did the above to improve readability.

Reply:
You're setting margin for all side as `0rem` and then setting it again margin left as `1.6rem` to overwrite the above styling. This shorthand just straight away defines all the sides (top, right, bottom, left). In my opinion it shortens the scss which also contributes to code readability. 

But nonetheless both of them works the same I guess 🤷🏼